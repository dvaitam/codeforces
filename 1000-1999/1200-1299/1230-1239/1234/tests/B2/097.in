26 10
17 10 8 18 14 15 15 17 18 10 6 17 20 17 10 19 7 10 5 1 11 4 14 13 17 6
