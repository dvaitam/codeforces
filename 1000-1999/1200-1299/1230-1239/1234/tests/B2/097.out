10
17 13 4 11 1 5 7 19 20 6
